src/main/java/org/prebid/server/auction/LatestRatesService.java

```diff
@@ -0,0 +1,50 @@
+package org.prebid.server.auction;
+
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.Json;
+import org.prebid.server.auction.model.Currency;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.Map;
+
+/**
+ * Class holding and updating latest currencies from server.
+ */
+public class LatestRatesService {
```

Comment:
I was thinking if functionality of this service really deserves to be in a separate service and had ended up with conclusion it might not. What about moving it to `CurrencyService`? It seems a pretty good fit there since currently these two services are actually work in tandem.